packages/components/CHANGELOG.md

```diff
@@ -37,7 +38,7 @@
 -   `withFocusReturn`: Refactor tests to `@testing-library/react` ([#45012](https://github.com/WordPress/gutenberg/pull/45012)).
 -   `ToolsPanel`: updated to satisfy `react/exhaustive-deps` eslint rule ([#45028](https://github.com/WordPress/gutenberg/pull/45028))
 -   `Tooltip`: updated to ignore `react/exhaustive-deps` eslint rule ([#45043](https://github.com/WordPress/gutenberg/pull/45043))
--   `SlotFill`: updated to satisfy `react/exhaustive-deps` eslint rule ([#44403](https://github.com/WordPress/gutenberg/pull/44403))
+-   `Context`: updated to ignore `react/exhaustive-deps` eslint rule ([#45044](https://github.com/WordPress/gutenberg/pull/45044))
```

Comment:
This entry should also be moved to the `Unreleased` section

Reply:
lol, I think I was noticing this and fixing it literally as you were leaving this comment.

Reply:
![image](https://user-images.githubusercontent.com/1083581/197031334-e0d6ee63-c80d-4555-8e4f-f72e84cc873a.png)